<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>235</width>
     <height>308</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_1log">
      <property name="placeholderText">
       <string>login</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_4name">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_2pass">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QRadioButton" name="radioButton_student">
      <property name="text">
       <string>student</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_3email">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEdit_5surname">
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="radioButton_teacher">
      <property name="text">
       <string>teacher</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButtonback_to_auth">
        <property name="text">
         <string>  &lt;-  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reg">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Registration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_note">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please, note: every field has a limit - 100 characters!</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_2pass</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>radioButton_student</tabstop>
  <tabstop>radioButton_teacher</tabstop>
  <tabstop>lineEdit_3email</tabstop>
  <tabstop>toolButtonback_to_auth</tabstop>
  <tabstop>pushButton_reg</tabstop>
  <tabstop>lineEdit_5surname</tabstop>
  <tabstop>lineEdit_4name</tabstop>
  <tabstop>lineEdit_1log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
